tests/CommunityToolkit.Aspire.Hosting.SurrealDb.Tests/AppHostTests.cs

```diff
@@ -0,0 +1,46 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using CommunityToolkit.Aspire.Testing;
+using Aspire.Components.Common.Tests;
+using FluentAssertions;
+using System.Net.Http.Json;
+
+namespace CommunityToolkit.Aspire.Hosting.SurrealDb.Tests;
+
+[RequiresDocker]
+public class AppHostTests(AspireIntegrationTestFixture<Projects.CommunityToolkit_Aspire_Hosting_SurrealDb_AppHost> fixture) : IClassFixture<AspireIntegrationTestFixture<Projects.CommunityToolkit_Aspire_Hosting_SurrealDb_AppHost>>
+{
+    [Fact]
+    public async Task SurrealResourceStartsAndRespondsOk()
+    {
+        const string resourceName = "db";
+        await fixture.ResourceNotificationService.WaitForResourceAsync(resourceName, KnownResourceStates.Running).WaitAsync(TimeSpan.FromMinutes(1));
```

Comment:
It might be more stable to wait until the resource is healthy rather than just Running, depending on how long it takes for SurrealDb to setup in the container and have the data seeded to it.

Reply:
`Healthy` is not an existing state. Do you have any example of that pattern?

Reply:
Ok. I found out how to do it.